.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -147,21 +141,13 @@ module.exports = (actionInfo) => {
       // to the correct versions
       await Promise.all(
         Array.from(pkgDatas.keys()).map(async (pkgName) => {
-          const { pkg, pkgPath, pkgData, packedPkgPath } = pkgDatas.get(pkgName)
-          // Copied from pnpm source: https://github.com/pnpm/pnpm/blob/5a5512f14c47f4778b8d2b6d957fb12c7ef40127/releasing/plugin-commands-publishing/src/pack.ts#L96
-          const tmpTarball = path.join(
-            pkgPath,
-            `${pkgData.name.replace('@', '').replace('/', '-')}-${
-              pkgData.version
-            }.tgz`
-          )
-          await execa('pnpm', ['pack'], {
+          const { pkgPath, packedPkgPath } = pkgDatas.get(pkgName)
+
+          await execa('yarn', ['pack', '-f', packedPkgPath], {
             cwd: pkgPath,
```

Comment:
At the very least, add the `COREPACK_ENABLE_STRICT=0` env var so that corepack wont error when running yarn